plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeMetadata.java

```diff
@@ -1908,20 +1909,46 @@ public void setTableProperties(ConnectorSession session, ConnectorTableHandle ta
                         "%s cannot be downgraded from %d to %d", WRITER_VERSION_PROPERTY, currentProtocolEntry.getMinWriterVersion(), writerVersion.get()));
             }
         }
+        long createdTime = Instant.now().toEpochMilli();
+
+        Map<String, String> existingConfiguration = handle.getMetadataEntry().getConfiguration();
```

Comment:
nit: I would inline this variable, it's not reused and it's needed only within the subsequent `if`.